.azure/templates/steps/setup_java.yaml

```diff
@@ -1,35 +1,12 @@
 # Step to setup JAVA on the agent
-# We use openjdkX, where X is Java version (e.g. 11 for OpenJDK 11).
 parameters:
-  - name: JDK_PATH
-    default: '/usr/lib/jvm/java-11-openjdk-amd64'
   - name: JDK_VERSION
     default: '11'
 
 steps:
-  - bash: |
-      sudo apt-get update
-    displayName: 'Update package list'
-  - bash: |
-      sudo apt-get install openjdk-11-jdk
-    displayName: 'Install openjdk11'
-    condition: eq(variables['JDK_VERSION'], '11')
-  - bash: |
-      sudo apt-get install openjdk-17-jdk
-    displayName: 'Install openjdk17'
-    condition: eq(variables['JDK_VERSION'], '17')
-  - bash: |
-      echo "##vso[task.setvariable variable=JAVA_VERSION_BUILD]11"
-      echo "##vso[task.setvariable variable=JAVA_VERSION]11"
-    displayName: 'Setup JAVA_VERSION=11'
-    condition: eq(variables['JDK_VERSION'], '11')
-  - bash: |
-      echo "##vso[task.setvariable variable=JAVA_VERSION_BUILD]17"
-      echo "##vso[task.setvariable variable=JAVA_VERSION]17"
-    displayName: 'Setup JAVA_VERSION=17'
-    condition: eq(variables['JDK_VERSION'], '17')
-  - bash: |
-      echo "##vso[task.setvariable variable=JAVA_HOME]$(JDK_PATH)"
-      echo "##vso[task.setvariable variable=JAVA_HOME__X64]$(JDK_PATH)"
-      echo "##vso[task.setvariable variable=PATH]$(jdk_path)/bin:$(PATH)"
-    displayName: 'Setup JAVA_HOME'
+  - task: JavaToolInstaller@0
+    inputs:
+      versionSpec: $(JDK_VERSION)
+      jdkArchitectureOption: 'x64'
+      jdkSourceOption: 'PreInstalled'
+      displayName: 'Configure Java'
```

Comment:
Is this misaligned? Should it be really inside `input`?

Reply:
Good spot, fixed!